packages/syft/src/syft/ast/klass.py

```diff
@@ -161,6 +161,87 @@ def run_class_method(
 
         return result
 
+    def run_smpc_class_method(
+        __self: Any,
+        *args: Tuple[Any, ...],
+        **kwargs: Any,
+    ) -> object:
+        """Run remote class method and get pointer to returned object.
+
+        Args:
+            *args: Args list of class method.
+            **kwargs: Keyword args of class method.
+
+        Returns:
+            Pointer to object returned by class method.
+        """
+        # stdlib
+        from uuid import UUID
```

Comment:
Q: Does it raise circular import error when kept file start?

Reply:
Hmm - it should not